dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/ApplicationProcessor.java

```diff
@@ -300,10 +304,14 @@ private OpenAPI generateModel(OpenApiConfig config, Container appContainer, WebM
         ClassLoader tccl = classLoadingService.createThreadContextClassLoader(appClassloader);
         Object oldClassLoader = THREAD_CONTEXT_ACCESSOR.pushContextClassLoaderForUnprivileged(tccl);
         try {
-            OpenApiStaticFile staticFile = StaticFileProcessor.getOpenAPIFile(appContainer);
-
             openAPIModel = OpenApiProcessor.modelFromReader(config, tccl);
-            openAPIModel = MergeUtil.merge(openAPIModel, OpenApiProcessor.modelFromStaticFile(staticFile));
+
```

Comment:
This block isn't all that easy to read and at least needs some comments. Personally I would tend to avoid those inline static calls ... can they be separated out at all?

Fair enough that you can only get the IOException when closing the file ... but maybe mention that the `staticFile` is an AutoClosable which will be closed as part of the try with resources?

Reply:
As discussed, I will add comments which relate this section of code back to the [processing rules in the spec](https://download.eclipse.org/microprofile/microprofile-open-api-3.1/microprofile-openapi-spec-3.1.html#_processing_rules)